lib/scheduler.js

```diff
@@ -54,23 +55,22 @@ class Scheduler {
         this.serverless.cli.log(`scheduler: scheduling ${fConfig.id}/${eventData.name} `
           + `with ${eventData.cron}`);
         schedule.scheduleJob(eventData.cron, () => {
-          const func = this._requireFunction(fConfig.id);
-          if (!func) {
+          const func = this._executeFunction(fConfig.id);
+          if (func === undefined) {
             this.serverless.cli.log(`scheduler: unable to find source for ${fConfig.id}`);
             return;
           }
-          this.serverless.cli.log(`scheduler: running scheduled job: ${fConfig.id}`);
-          func(
-            this._getEvent(eventData),
-            this._getContext(fConfig),
-            () => {}
-          );
+          this.serverless.cli.log(`scheduler: Succesfully run scheduled job: ${fConfig.id}`);
         });
       }
     }
     return BbPromise.resolve();
   }
 
+  _executeFunction(fName) {
+    return execSync(`serverless invoke local --function ${fName} --data {"ping":"ping"}`, {cwd: './', stdio: 'inherit' });
+  }
+  
   _requireFunction(fName) {
```

Comment:
It seems as if `_requireFunction` is no longer being used.  Can you remove it?  Also, please update any tests that are using this.

Reply:
I do need to do a clean up. Can close this pr and add a bit more functionality and the testingn testing you requested before reopening